Tools/scripts/build-jsbsim.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
 
 set -e
-set -x
```

Comment:
Unrelated; please remove.

Reply:
Done
But I really think that this should be removed. Maybe in a new PR?